app/workers/ingest_latest_blocked_lane_cases_worker.rb

```diff
@@ -18,5 +20,7 @@ def perform
 
     # 3. Ingest the records retrieved in step 2:
     Sf311Case.ingest_csv_case_data!(blocked_lane_cases_csv)
+
+    Rails.logger.info("Successfully updated 311 cases")
```

Comment:
It might be useful to log the count of records ingested as well.

Reply:
Good idea! Added the ingested record count to the log message.